pithos/plugins/notify.py

```diff
@@ -45,6 +45,23 @@ def on_prepare(self):
         if 'action-icons' in caps:
             self.notification.set_hint('action-icons', GLib.Variant('b', True))
 
+        # GNOME Shell 3.20 or higher has bultin MPRIS functionality that makes
+        # persistent song notifications redundant.
+        has_built_in_mpris = False
+        if server_info['name'] == 'gnome-shell':
+            version = server_info['version'].split('.')
+            if version and len(version) >= 2:
+                try:
+                    major_version = int(version[0])
+                    minor_version = int(version[1])
+                except ValueError:
+                    major_version = minor_version = 0
+                if major_version >= 3 and minor_version >= 20:
```

Comment:
I can make 4.0 equal true but I don't know if that would be more harm than good really. The way GNOME changes things from release to release even in minor versions who knows what 4.0 will actually mean?